dashboard/app/templates/templates.html

```diff
@@ -87,36 +87,30 @@ <h1><a href="/{{$.Namespace}}">syzbot</a></h1>
           </div>
         {{end}}
 
-        <div class="navigation_tab{{if eq .URLPath (printf "/%v/graph/bugs" $.Namespace)}}_selected{{end}}">
-          <a href='/{{$.Namespace}}/graph/bugs'>
-            <span style="color:DarkOrange;">📈</span> Kernel Health</a>
-        </div>
-
-        <div class="navigation_tab{{if eq .URLPath (printf "/%v/graph/found-bugs" $.Namespace)}}_selected{{end}}">
-          <a href='/{{$.Namespace}}/graph/found-bugs'>
-            <span style="color:DarkOrange;">📈</span> Bugs/Month</a>
-        </div>
-
-        <div class="navigation_tab{{if eq .URLPath (printf "/%v/graph/lifetimes" $.Namespace)}}_selected{{end}}">
-          <a href='/{{$.Namespace}}/graph/lifetimes'>
-            <span style="color:DarkOrange;">📈</span> Bug Lifetimes</a>
-        </div>
-
-        <div class="navigation_tab{{if eq .URLPath (printf "/%v/graph/fuzzing" $.Namespace)}}_selected{{end}}">
-          <a href='/{{$.Namespace}}/graph/fuzzing'>
-            <span style="color:DarkOrange;">📈</span> Fuzzing</a>
+        <div class="dropdown navigation_tab">
+          <button class="dropbtn"><span style="color:DarkOrange;">📈</span>Graphs</button>
+          <div class="dropdown-content">
+            <a  class="navigation_tab{{if eq .URLPath (printf "/%v/graph/bugs" $.Namespace)}}_selected{{end}}"
+                href='/{{$.Namespace}}/graph/bugs'>Kernel&nbsp;Health</a>
+            <a  class="navigation_tab{{if eq .URLPath (printf "/%v/graph/found-bugs" $.Namespace)}}_selected{{end}}"
+                href='/{{$.Namespace}}/graph/found-bugs'>Bugs/Month</a>
+            <a  class="navigation_tab{{if eq .URLPath (printf "/%v/graph/lifetimes" $.Namespace)}}_selected{{end}}"
+                href='/{{$.Namespace}}/graph/lifetimes'>Bug&nbsp;Lifetimes</a>
+            <a  class="navigation_tab{{if eq .URLPath (printf "/%v/graph/fuzzing" $.Namespace)}}_selected{{end}}"
+                href='/{{$.Namespace}}/graph/fuzzing'>Fuzzing</a>
+            <a  class="navigation_tab{{if eq .URLPath (printf "/%v/graph/crashes" $.Namespace)}}_selected{{end}}"
+                href='/{{$.Namespace}}/graph/crashes'>Crashes</a>
```

Comment:
This one is technically not a graph

Reply:
Yeah. Some names polishing will be needed.